sites/diverseeducation.com/config/navigation.js

```diff
@@ -79,7 +80,7 @@ module.exports = {
       { href: 'https://top100.diverseeducation.com', label: 'Top 100', target: '_blank' },
       { href: 'https://mediakit.diverseeducation.com/', label: 'Advertise', target: '_blank' },
       { href: 'https://diversejobs.net', label: 'Jobs', target: '_blank' },
-      // { href: '#', label: 'Shop', target: '_blank' },
+      { href: 'http://shop.diverseeducation.com/wordpress/', label: 'Shop', target: '_blank' },
```

Comment:
@jwade1327 - this should be using `https`.  I sent a note to Marcos; will need to wait for him to set that up.